src/commandsAndMenu.tsx

```diff
@@ -716,11 +716,11 @@ export function addCommands(
       const extensions = files
         .map(file => PathExt.extname(file.to))
         .filter(extension => extension.length > 0);
-      const subject = extensions.length > 1 ? 'extensions' : 'extension';
-      return trans.__(
-        'Ignore %1 %2 (add to .gitignore)',
-        extensions.join(', '),
-        trans.__(subject)
+      return trans._n(
+        'Ignore %2 extension (add to .gitignore)',
+        'Ignore %2 extensions (add to .gitignore)',
```

Comment:
This should probably be a `%1` ?

Reply:
using %1 was actually picking up the number arg (extensions.length). I was testing with this: https://github.com/JessicaBarh/jupyterlab-fr_FR/tree/handle-plurals/jupyterlab_language_pack_fr_FR/extensions/jupyterlab-git

Reply:
You are absolutely right! My bad